toonz/sources/include/tsound.h

```diff
@@ -248,6 +248,7 @@ Range in seconds
 class TSoundDeviceException final : public TException {
 public:
   enum Type {
+    NoDevice,           // no device found
```

Comment:
Hello,
Thanks @RodneyBaker and @turtletooth for your review.
I have not checked that the enum reordering has no side effect. Actualy I did not think it could have one unless someone use the values rather than the names of the enum.